README.md

```diff
@@ -136,6 +136,19 @@ First - ensure that the following environment variables are set to the appropria
 
 NOTE: These values (and keys) need to be shared/common across all microservices that use each common authenticator host. If any of the environment *keys* for each of these attributes needs to be modified these can be reconfigured in fsd_utils/authentication/config.py.
 
+To generate new keys, you use the following commands:
```

Comment:
```suggestion
To generate new keys, you use the following commands:
(Note: be aware of where your keys get created, when setting these keys as envvars in cf you'll need to be in the directory where you created them).
```

Reply:
Good shout, added in https://github.com/communitiesuk/funding-service-design-utils/pull/76/commits/66d905b02d5ae57787d53d94e1fa18686797fab5 👍